.gitignore

```diff
@@ -83,12 +83,12 @@ celerybeat-schedule
 
 # Environments
 .env
-.venv
+# .venv
 env/
-venv/
+# venv/
 ENV/
 env.bak/
-venv.bak/
+# venv.bak/
```

Comment:
committing commented stuff! is this what we learned?

Reply:
it was a quick fix...i can fix it